Kwf/Srpc/Client.php

```diff
@@ -62,7 +62,8 @@ protected function _performRequest(array $params)
     {
         $httpClientConfig = array(
             'timeout' => $this->_timeout,
-            'persistent' => true
+            'persistent' => true,
+            'adapter' => 'Zend_Http_Client_Adapter_Curl'
```

Comment:
sobald proxy verwendet wird funktioniert das ganze wieder nicht mehr, gleich machen wie in getMediaOutputForDimension bei Kwf_Media_Output_Component

Reply:
warum nicht? proxy wird unten überschrieben und der erbt von socket was wiederum das gleiche problem verursachen würde?

Reply:
hast recht, sorry